### PR TITLE
fix(tabs): Show first tab when set to active

### PR DESCRIPTION
**Pull request by tmorehouse**
May address issue #911

### CODE REVIEW

File: lib/components/tab.vue [hunk 1]
```diff
@@ -36,7 +36,7 @@
         data() {
             return {
                 localActive: this.active && !this.disabled,
-                show: false
+                show: Boolean(this.active && !this.disabled)
```
Comment (mosinve) [inline lib/components/tab.vue]: why not just localActive?

Comment (tmorehouse) [inline lib/components/tab.vue]: We don't have access to `this.localActive` yet at this stage, as it hasn't been returned to the VM.

Comment (mosinve) [inline lib/components/tab.vue]: check this https://codepen.io/cih/pen/eErKvm

Comment (mosinve) [inline lib/components/tab.vue]: ah.... but then at mounted? i dont like this similar code.....

**Comment by mosinve** [comment]
Troy... i thought... isn't localActive supposed to be a computed value? if it's a data - then it will be valuated only once.. upon creation of component

**Comment by tmorehouse** [comment]
b-tabs sets the `localActive` data in `b-tab`, as it controls which tabs are visible

**Comment by mosinve** [comment]
but computed vars may have a setter..

**Comment by tmorehouse** [comment]
b-tabs still need to be able to control which tab is active through:
https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/lib/components/tabs.vue#L189-L196

File: lib/components/tab.vue (hunk 1 above)
Comment (tmorehouse) [inline lib/components/tab.vue]: Hmmm... mounted may work, although it should maybe look at localActive on mounted (depending on if b-tabs is set to `lazy` or not.

**Comment by mosinve** [comment]
and if we assign active prop not at design, but at runtime? will it handle it?

**Comment by tmorehouse** [comment]
Someone can manually set active on all their tabs, but we ensure that only one tab should be visible at a time (we select the last 'active" tab found).

maybe show should be a computed prop (or an `inShow` computed prop that looks at the `show` data and the localActive data?

**Comment by mosinve** [comment]
tabs not dynamically following tab's `active` prop. it looks at localActive, which sets up at mount... so if we set `tab.active` at runtime - this wont work.

**Comment by tmorehouse** [comment]
I tried setting active tab on fiirst tab (with `currentTab` set to index 0, and the tab is there, but missing the `show` property.,

**Comment by mosinve** [comment]
at runtime?

**Comment by mosinve** [comment]
let's try with computed show, i'll check all my doubts at local playground...

**Comment by tmorehouse** [comment]
In the b-tabs doc page.

I set first tab as active (with active prop), and left the app data `tabIndex` as 0

Tab is definitely showing, but with opacity still at 0 because the `show` class was missing on the active tab.

It appears the first `active` tab  isn't triggering the transition triggers, which then add the `show` cl;ass to the `b-tab`

**Comment by tmorehouse** [comment]
You can see above, that the tab is `active` but missing the `show` class, so opacity is still `0`

**Comment by mosinve** [comment]
Ah... I saw this.. and after that i tried to set show at true ... and so on... 
But it wont help in case active will set true at runtime...

**Comment by tmorehouse** [comment]
We could change the css transition sniffing and set a duration on the transition to 150ms (if fade, 0 if noFade)

**Pull request merged**